dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperationServer.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// REST API Operation Server.
+/// </summary>
+public sealed class RestApiOperationServer
```

Comment:
Is this OpenApi terminology?  If not, would service make more sense?  I'm guessing here we are not referencing a domain/server here but rather a specific service?